hack/e2e-suite/update.sh

```diff
@@ -124,10 +122,15 @@ function teardown() {
   echo "Cleaning up test artifacts"
   ${KUBECTL} stop rc update-demo-kitten || true
   ${KUBECTL} stop rc update-demo-nautilus || true
+  kill -TERM "${kubectl_proxy_pid:-}" &> /dev/null || true
 }
 
 trap "teardown" EXIT
 
+# Launch a local proxy to the apiserver
+${KUBECTL} proxy --port=8011 &
```

Comment:
For the e2e tests I used a non-default port for proxying. When I originally tried using the default port, I saw a bunch of "Failed to bind to port 8001" in the logs.